src/effects/native/phasereffect.cpp

```diff
@@ -173,7 +192,7 @@ void PhaserEffect::processChannel(const ChannelHandle& handle,
         right = processSample(right, oldInRight, oldOutRight, filterCoefRight, stages);
 
         // Computing output combining the original and processed sample
-        pOutput[i] = pInput[i] * (1.0 - 0.5 * depth) + left * depth * 0.5;
-        pOutput[i + 1] = pInput[i + 1] * (1.0 - 0.5 * depth) + right * depth * 0.5;
+        pOutput[i] = pInput[i] + left * depth * 0.5;
+        pOutput[i + 1] = pInput[i + 1] + right * depth * 0.5;
```

Comment:
I do not understand your proposal here. I have added a new commit turning this into a send effect. I think it works well. Please test this approach.